src/content/whats-new/2020/07/overall-new-relic-one-experience-updates.md

```diff
@@ -4,7 +4,7 @@ summary: We've made updates across all of New Relic One!
 releaseDate: '2020-07-30'
 ---
 
-We listened to you about what’s working and what’s not. That’s why we [reimagined New Relic One](https://blog.newrelic.com/product-news/reimagined-new-relic-one-experience/) by unifying all of our capabilities within a single experience, completely removing the barriers to observability. Easily navigate the platform and take advantage of powerful, new and improved features like search, a feature toggle, an Alerts & AI section, a feedback form, the ability to add data, and more!
+We listened to you about what’s working and what’s not. That’s why we [reimagined New Relic One](https://blog.newrelic.com/product-news/reimagined-new-relic-one-experience/) by unifying all of our capabilities within a single experience, completely removing the barriers to observability. Easily navigate the platform and take advantage of powerful, new and improved features like search, a feature toggle, an Alerts section, a feedback form, the ability to add data, and more!
```

Comment:
@homelessbirds is it appropriate to change these historical references since we did refer to applied intelligence back in 2020?